### DISCUSSION TITLE
Disable Leds on Megadesk Companion

### DISCUSSION THREAD

**Discussion by andreassave** [gcormier/megadesk#155] [Q&A]
I just received and installed Megadesk and Megadesk Companion (using Home Assistant)! Seems to work as expected which is great!

There are two leds on the Megadesk Companion that are on/bright all the time. Is there any way I can disable the leds?

**Comment by gcormier** [comment]
Hello! Unfortunately not. You could de-solder them if you have the capability to do so. You could also cut the traces with a utility knife as well - I can post instructions on this if you would like.

Note these were removed in the most recent version due to feedback.

**Comment by andreassave** [comment]
Thanks for fast answer. Instruction would be helpful. I might be able to de-solder if it is not too complicated.

**Reply by gcormier** [reply]
![image](https://github.com/gcormier/megadesk/assets/8388983/b4080fbd-cbb5-46dd-a2e0-3b361f7ad031)
![image](https://github.com/gcormier/megadesk/assets/8388983/0dc2c866-36d0-414d-b8a8-b494c0fec15f)


1. You can cut the traces at the pink lines.
2. You could de-solder the resistors that are circled. As they are tiny, you probably only need to apply heat to 1 side with a bit of fresh solder and the entire resistor should come off after 3-4 seconds.
3. You could also desolder the LED themselves, but the resistors above will be much easier and accomplish the same goal.
